flambe/learn/train.py

```diff
@@ -45,7 +46,9 @@ def __init__(self,
                  lower_is_better: bool = False,
                  max_grad_norm: Optional[float] = None,
                  max_grad_abs_val: Optional[float] = None,
-                 extra_validation_metrics: Optional[Dict[str, Metric]] = None) -> None:
+                 extra_validation_metrics: Optional[Union[Dict[str, Metric], Iterable[Metric]]] = None,
+                 extra_training_metrics: Optional[Union[Dict[str, Metric], Iterable[Metric]]] = None,
+                 extra_training_metrics_log_interval: int = -1) -> None:
```

Comment:
doesn't need to be just for extras. Maybe let's treat the log interval for training metrics AND loss?

Reply:
-1 is also a weird default. Please use None.

Reply:
or 1 lol since that is currently the actual default